apps/web/vibes/2px/examples/date-picker.tsx

```diff
@@ -0,0 +1,12 @@
+import DatePicker from '@/vibes/2px/components/date-picker'
+
+export default function Preview() {
+  return (
+    <div className="flex max-w-lg flex-col items-center justify-center gap-5 bg-background p-5 sm:min-h-64 sm:p-8 lg:min-h-[500px] lg:p-12">
+      <DatePicker />
+      <DatePicker disabled />
```

Comment:
Since the popup is rendered at the top, I suggest starting with the disabled version at the top to give some more room for the enabled date-pickers' calendars.